core/generated-sources/src/test/kotlin/org/jetbrains/kotlinx/dataframe/samples/api/Access.kt

```diff
@@ -630,6 +630,7 @@ class Access : TestBase() {
         df.select { age and name }
         df.fillNaNs { dfsOf<Double>() }.withZero()
         df.remove { cols { it.hasNulls() } }
+        df.group { cols { it.data != name } }.into { "nameless" }
```

Comment:
i don't understand this line

Reply:
It used to be:
```kotlin
df.group { cols { it != name } }.into { "nameless" }
```
because `cols` took a `(AnyCol) -> Boolean`. However, I converted the argument to a `ColumnFilter<*>` which is like `(ColumnWithPath<*>) -> Boolean`. This also in preparation for being able to do `cols {}.recursively()` (Since `dfs {}` also works with columns with path).
However, since the column is now wrapped into a ColumnWithPath ,you need to call `.data.` to get the actual column and be able to compare it with some accessor. If you have a better idea how to solve this, let me know :)

Reply:
Can you mention it in that issue? And, how that `it.hasNulls()` above works?

Reply:
ColumnWithPath still inherits from DataColumn, so the only thing changing is exact equality

Reply:
Ok, then let's continue there later. Not a show stopper

Reply:
yeah, we'll get back to it in the dfs renaming issue :)